introduction.md

```diff
@@ -0,0 +1 @@
+I am a mobile applications developer.  I have been developing for 6+ years.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.